src/libs/kata-types/src/mount.rs

```diff
@@ -68,6 +68,19 @@ pub fn is_kata_host_dir_volume(ty: &str) -> bool {
     ty == KATA_HOST_DIR_VOLUME_TYPE
 }
 
+/// nydus extraoptions
```

Comment:
I think it's better to add a space between extra and options.

Reply:
fixed